packages/core/src/Dialog/Dialog.styled.tsx

```diff
@@ -254,7 +255,7 @@ export const DialogContent = ({
               )}
             </SmoothTransitionBox>
           ) : (
-            <Box height='100%' style={{ overflowY: 'scroll' }}>
+            <Box height='100%' style={innerContentScroll && { overflowY: 'scroll' }}>
```

Comment:
Using `&&` while assigning prop values is not the best solution. In this context if `innerContentScroll` is falsy, then `style` would be set to `false`. You can easily see that error in VS Code as `false` is not a valid value for `style` 
<img width="351" alt="image" src="https://github.com/priceline/design-system/assets/5671981/c2cdba3d-cf2a-4178-a205-d850db1f26bc">

Reply:
My editor did not throw that error for me 🤔, pushing a fix for that now.